### DISCUSSION TITLE
Bun <3 Remult?

### DISCUSSION THREAD

**Discussion by SpeedoPasanen** [noam-honig/radweb#236] [Q&A]
Anyone tried to run Remult on Bun yet? Works out of the box? :) Because bun just went 1.0 I'm considering switching.

**Comment by jycouet** [comment]
Using SvelteKit & Remult, I just did the test and it works 🎉!

```bash
# Install bun on your machine (WSL for me)
curl -fsSL https://bun.sh/install | bash

# install dep (1.1sec (1.6sec in pnpm))
bun i

# start  the  project with bun instead of nodejs
bun --bun run dev
# => I see some postgres queries and UI updates 🎉🎉🎉

# update adapter in `svelte.config.js`
import adapter from "svelte-adapter-bun";

# build (6.2 sec (7.9 sec in pnpm)
bun run build

# run production
cd build && bun run start
# => I see some postgres queries and UI updates 🎉🎉🎉
```

Bun is quite early, and it's not a big test, but for what I saw, it seems to work well 🎉🎉🎉

FYI: 
-  `repo.getEntityRef(u[0]).metadata.caption` is working in prod as well. 👍
- trying to update on a larger code base didn't work so far (workspace issue atm).
